packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -548,7 +546,7 @@ export class ApexDebug extends LoggingDebugSession {
 
   private lock = new AsyncLock({ timeout: DEFAULT_LOCK_TIMEOUT_MS });
 
-  protected idleTimers: NodeJS.Timer[] = [];
+  protected idleTimers: Array<ReturnType<typeof setTimeout>> = [];
```

Comment:
another spot where jest tests make a different.  This was the recommending type for NodeJS.Timer.